Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -659,48 +668,76 @@ bool LineToLineIntersectionHelper(carve::geom::vector<2>& v1,
 }
 
 /**********************************************************************************************/
-
-bool GeomUtils::LineSegmentToLineIntersection(carve::geom::vector<2>& v1,
-											  carve::geom::vector<2>& v2, 
-											  carve::geom::vector<2>& v3,
-											  carve::geom::vector<2>& v4,
-										std::vector<carve::geom::vector<2> >& result )
+/*! \brief Calculates coordinates of the intersection point.
+* \param[in] initialPointSeg1		Initial point of the first intersecting segment.
+* \param[in] terminalPointSeg1		Terminal point of the first intersecting segment.
+* \param[in] initialPointSeg2		Initial point of the second intersecting segment, which describes the direction of the line.
+* \param[in] terminalPointSeg2		Terminal point of the second intersecting segment, which describes the direction of the line.
+* \param[out] intersectionPoint		Coordinates of the intersection point between first line segment and second line.
+*/
+bool GeomUtils::LineSegmentToLineIntersection(const carve::geom::vector<2>& initialPointSeg1,
+												const carve::geom::vector<2>& terminalPointSeg1, 
+												const carve::geom::vector<2>& initialPointSeg2,
+												const carve::geom::vector<2>& terminalPointSeg2,
+	carve::geom::vector<2> & intersectionPoint)

```

Comment:
- missing `\return` explanation
- please make sure, that the indentation is consistent

Reply:
Resolved:  a619817 and 8f80eb1